### ISSUE TITLE
App update process should not be totally silent for user

### ISSUE THREAD

**Issue by ghost** [cozy/cozy-home-v2#246]
I recently experienced this as a goofyUser:
- went to the app manager page
- chose to update my cozymail clicking on the appropriate button
- received an information popup asking for various permissions
- clicked to confirm I wanted to update
- information popup disappears
- **nothing**
  - no popup message telling me "update in progress…"
  - no popup message telling me "{appname} has been updated successfully"
    It is only when I realized some details changed in the app that I understood update was successful, I thought it failed silently.

note that I dont object _automatic_ silent updates, I suggest process should dump some hints to the user when he/she decides to update _manually_.

**Comment by poupotte** [comment]
@jsilvestre, @goofy-bz Is it still relevant with the new home design ?

**Comment by jsilvestre** [comment]
It could still be improved, I believe. The behaviour is pretty much the same, even if it doesn't feel buggy like it used too.

**Comment by Zykino** [comment]
Yep having something like a progress bar added in the modal saying something like "Updating app X, n/m" would be great.
Maybe even having the list of everything to update that's disable itself would be a big plus.

**Comment by clochix** [comment]
On Rasperry Pi, updates can take really a long time without feedback, so having some progress indicator would improve users experience.
